default/stringtables/en.txt

```diff
@@ -1974,6 +1974,9 @@ Enable autosave when resigning or closing the game. In multiplayer hostless mode
 OPTIONS_DB_AUTOSAVE_HOSTLESS
 Enable autosave in hostless mode.
 
+OPTIONS_DB_AUTOSAVE_HOSTLESS_EACH_PLAYER
+Enable autosave in hostless mode after each player orders.
```

Comment:
meaning unclear... rephrase / be more specific

Reply:
I've tried.